contracts/SL2RD.sol

```diff
@@ -62,21 +83,47 @@ contract SL2RD is
     /// owner for each tokenId. This is prefilled with the owner's
     /// address for each token, but designed with flexibilty to include
     /// others during initialization if they are known.
+    /// The communitySplitsBasisPoints is a const percentage denoted in
+    /// basis points for how many of the slots the creator initially wants
+    /// to allocate for everyone else.
+    /// @param addresses_ The addresses for initial distribution.
+    /// @param tokenIds_ The tokenIds for initial distribution.
+    /// @param communitySplitsBasisPoints_ The percentage of slots allocated
```

Comment:
Can you give an example here of what basis point value would be supplied for a 20% community distribution?